kolibri/core/content/management/commands/importcontent.py

```diff
@@ -377,7 +397,9 @@ def _transfer(  # noqa: max-complexity=16
                             f = files_to_download.pop()
                             filename = get_content_file_name(f)
                             try:
-                                dest = paths.get_content_storage_file_path(filename)
+                                dest = paths.get_content_storage_file_path(
+                                    filename, contentfolder=content_dir
```

Comment:
Nit: maybe it's better to use the same variable name everywhere, eg renaming the new CLI parameter to `--content-folder`. Or do you prefer to use "content dir" as the exposed name and "contentfolder" as the internal name?

Reply:
`CONTENT_DIR` is the configuration option and it's used in other places, so I think it's better to use `content-dir` for this option.